image_definitions/base/Dockerfile

```diff
@@ -1,88 +1,119 @@
-ARG IMAGE_TAG=7.3-alpine-3.14
-
-# Pull the base image to use
-FROM mcr.microsoft.com/powershell:${IMAGE_TAG}
-
 # Add the arguments for the apps to install in the base image
 ARG TERRAFORM_VERSION=1.5.1
-ARG AMIDOBUILD=v0.0.230
+ARG AMIDOBUILD=v0.0.249
 ARG TASKCTL_VERSION=1.4.2
 ARG KUBE_VERSION=v1.23.14
 ARG AZURE_CLI_VERSION=2.48.1
 ARG AZURE_AZ_MODULE_VERSON=10.0.0
-ARG AWS_CLI_VERSION=2.12.4
+ARG AWS_CLI_VERSION=2.13.21
```

Comment:
This AWS_CLI_VERSION ARG isn't being used so can be removed

Reply:
It gets used on line 21, but I see I've duplicated the argument below with an older version 😂